src/coreclr/vm/dllimport.cpp

```diff
@@ -2548,6 +2548,22 @@ namespace
         _ASSERTE(*nlt == nltAnsi || *nlt == nltUnicode);
         return S_OK;
     }
+
+    HRESULT ParseCallingConventionFromAttributeConstructor(_Inout_ CustomAttributeParser& ca, _Out_ CorInfoCallConvExtension* callConv)
+    {
+        LIMITED_METHOD_CONTRACT;
+        _ASSERTE(callConv != NULL);
+
+        CaArg callConvArg;
+        callConvArg.InitEnum(SERIALIZATION_TYPE_I4, (ULONG)0);
+        HRESULT hr = ParseKnownCaArgs(ca, &callConvArg, 1);
```

Comment:
I believe this can return `S_FALSE`.

Reply:
I don't see where?
All our existing usage also seems to just check for failed.

Reply:
Oh sorry. I was confused about the finding a Custom Attribute logic and parsing. You're correct.